app/controllers/concerns/authorization_request_management.rb

```diff
@@ -0,0 +1,27 @@
+module AuthorizationRequestManagement
```

Comment:
```suggestion
module AuthorizationRequestsManagement
```

au pluriel pour être consistant avec le reste.